<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimuRéac</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="simulationBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="simulation">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>479</width>
            <height>517</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="elementsBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="Formule">
            <property name="text">
             <string>Formule</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Concentration">
            <property name="text">
             <string>.L⁻¹</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Logo">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>IMG</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="elements">
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>0</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>253</width>
            <height>303</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Pause">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="Remplissages" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="Burette">
            <property name="text">
             <string>Remplir Burette</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Becher">
            <property name="text">
             <string>Remplir Becher</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Eau">
            <property name="text">
             <string>Ajouter/Retirer Eau pure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Temperature">
            <property name="text">
             <string>Changer Température Becher</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionChargerSimulation"/>
    <addaction name="actionSauvegarderSimulation"/>
    <addaction name="actionImporterParticule"/>
    <addaction name="actionSauvegarderParticule"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionEditer"/>
    <addaction name="actionFermer"/>
   </widget>
   <widget class="QMenu" name="menuInstruments">
    <property name="title">
     <string>Instruments</string>
    </property>
    <addaction name="actionChronometre"/>
    <addaction name="actionThermometre"/>
    <addaction name="actionPHMetre"/>
    <addaction name="actionConductimetre"/>
    <addaction name="actionColorimetre"/>
    <addaction name="actionOrdinateur"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionSombre"/>
    <addaction name="actionFond"/>
    <addaction name="actionCompteurFPS"/>
    <addaction name="actionPleinEcran"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="actionAPropos"/>
    <addaction name="actionDonnations"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuInstruments"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEditer">
   <property name="text">
    <string>Editer la simulation</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionThermometre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Thermomètre</string>
   </property>
  </action>
  <action name="actionPHMetre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>pHMètre</string>
   </property>
  </action>
  <action name="actionConductimetre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Conductimètre</string>
   </property>
  </action>
  <action name="actionChronometre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chronomètre</string>
   </property>
  </action>
  <action name="actionSombre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Thème sombre</string>
   </property>
  </action>
  <action name="actionFond">
   <property name="text">
    <string>Couleur d'arrière-plan</string>
   </property>
  </action>
  <action name="actionColorimetre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Colorimètre</string>
   </property>
  </action>
  <action name="actionCompteurFPS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Compteur FPS</string>
   </property>
  </action>
  <action name="actionImporterParticule">
   <property name="text">
    <string>Importer Particule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionChargerSimulation">
   <property name="text">
    <string>Charger Simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSauvegarderParticule">
   <property name="text">
    <string>Sauvegarder Particule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSauvegarderSimulation">
   <property name="text">
    <string>Sauvegarder Simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer la simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOrdinateur">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ordinateur</string>
   </property>
  </action>
  <action name="actionPleinEcran">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plein écran</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Aide</string>
   </property>
  </action>
  <action name="actionAPropos">
   <property name="text">
    <string>À propos</string>
   </property>
  </action>
  <action name="actionDonnations">
   <property name="text">
    <string>Donations</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
